### PR TITLE
feat(inventory-fertiliser-transaction): add resource documentation

### PR DESCRIPTION
**Pull request by ryanjosephtorrado**
Ticket: [AB#21866](https://dev.azure.com/rezaresystems/15020b92-fee6-4202-956f-9036d1e1da84/_workitems/edit/21866)

----

Notes

----

## PR Checklist
- [ ] Documentation has been checked by at least one team member
- [x] Links work to navigate documentation
- [ ] Images stored in `_images`
- [x] Spell checked

> If some of the above are not completed why?
> **REASON HERE**

### CODE REVIEW

File: docs/resource-types/inventory/fertiliser-transaction.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+|-|-|-|
+|**Content Type**|The type of the referenced entity|String|
+|**Related**|A URL that may be used to retrieve the related resource|String|
+|**Relationship**|How the referenced resource relates to the current resource|String|
+|**Active Ingredients**|Describes an ingredient of a product|Array of [Active Ingredients](#active-ngredients)|
+|**Analysis**|Used to specify analysis of a product|Array of [Analysis](#analysis)|
+|**Brand**|The brand of the product|String|
+|**Claims**|The claims of the product|Array of [Identifier](/resource-types/common.md#identifier)|
+|**Crop**|Defines a managed crop, or a species in a mixed sward|[Crop](#crop)|
+|**form**|Product physical form|Enumeration|
```
Comment (sb226) [inline docs/resource-types/inventory/fertiliser-transaction.md:143]: Can you make first letter Capital please :p

Comment (ryanjosephtorrado) [reply docs/resource-types/inventory/fertiliser-transaction.md:143]: Absolutely! 😄

File: docs/resource-types/inventory/fertiliser-transaction.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+|**Transaction Kind**|Identifies the transaction kind<br>(Valid values: `Receipt, Disposal, OnHand, StockTake, Use`) |Enumeration|
+|**Quantity**|The overall volume, weight or count of the product in the transaction in the units defined| String| 
+|**Units**|The units of the quantity specified|String|
+|**Supplier Name**|The supplier of the product in this transaction |String|
+|**Expiry Date**|A particular point in the progression of time|Datetime| 
+|**Total Cost**|Total cost applied to this transaction |Number|
+|**Currency**|The currency of the cost|String|
+|**Pack Size**|The volume or weight of the product in a pack in the units defined|Number|
+|**Number Of Packs**|The number of packs of the product in the transaction |Number|
+|**Product**|The fertiliser product in this inventory transaction|[Product](#product)|
```
Comment (sb226) [inline docs/resource-types/inventory/fertiliser-transaction.md:106]: There also this property called Link, is that something that we can ignore? 
image

Comment (ryanjosephtorrado) [reply docs/resource-types/inventory/fertiliser-transaction.md:106]: Yes, we can ignore that

**Review by sb226 — APPROVED** [review]
Get it outta here...!
yeet-lion-king

**Pull request merged**